manifests/init.pp

```diff
@@ -221,6 +221,11 @@
 #       RPM: https://yum.datadoghq.com/stable/7/x86_64/ (with matching agent version and architecture)
 #       Windows: https://https://s3.amazonaws.com/ddagent-windows-stable/
 #       String. Default: undef
+#   $rpm_repo_gpgcheck
+#       Whether or not to perform repodata signature check for RPM repositories.
+#       Applies to Red Hat and SUSE platforms. When set to `undef`, this is activated
+#       for all Agent versions other than 5 when `agent_repo_uri` is also undefinded.
```

Comment:
Because most people don't sign their RPM repository metadata and if people are using custom repos where they mirror agent packages, it's very likely that trying to do `repo_gpgcheck` on these would fail (and could be considered a backwards incompatible change).